app/presenters/nsm/check_answers/cost_summary_card.rb

```diff
@@ -138,17 +140,28 @@ def vat_rate
         @vat_rate ||= claim.firm_office.vat_registered == 'yes' ? Pricing.for(claim).vat : 0.0
       end
 
-      def format(value, formatted: true)
+      def format(value, formatted: true, accessibility_key: '')
         return value unless formatted
         return nil if value.nil?
 
-        { text: NumberTo.pounds(value), numeric: true }
+        if accessibility_key.present?
+          { text: accessible_text(accessibility_key, NumberTo.pounds(value)), numeric: true }
+        else
+          { text: NumberTo.pounds(value), numeric: true }
+        end
+      end
+
+      def accessible_text(key, value)
+        scope = show_adjustments ? 'with_adjustments' : 'base'
+
+        text = I18n.t("nsm.steps.check_answers.groups.cost_summary.#{scope}.accessibility.#{key}")
+        sanitize("#{govuk_visually_hidden(text)} #{value}")
       end
 
       def t(key, numeric: true, width: nil)
         scope = show_adjustments ? 'with_adjustments' : 'base'
         {
-          text: I18n.t("nsm.steps.check_answers.groups.cost_summary.#{scope}.#{key}"),
+          text: sanitize(I18n.t("nsm.steps.check_answers.groups.cost_summary.#{scope}.#{key}")),
```

Comment:
This comes out of requirement to either use an empty <td> for the 1:1 cell (used to be "Item") or put visually hidden content in it. govuk_table does not allow <td> inside a thead [even if you specify `header: false` for the cell]. An empty <th> fails accessibility tests (although oddly other places where there is an empty <th> are not being caught by the accessibility checks it seems). The i18n value is html to put a govuk-visually-hidden span in, so it needs sanitization, despite it being called `items_html` 🤷

Reply:
So the issue is that apparently I18n.t doesn't automatically mark "_html" key outputs as HTML-safe, even though the `t` helper in views and controllers does. 

BUT that's all it's doing, so rather than `sanitize` which parses the string and tries to scrub out illegal chars, you can simply call `.html_safe` to mark it as safe, because you know it is safe.

Reply:
yep, `html_safe`  works. changed that